cmd/root.go

```diff
@@ -33,6 +33,18 @@ var rootCmd = &cobra.Command{
 
 const globalTimeout = 5 * time.Minute
 
+var arguments = struct {
+	policyConfiguration string
+	imageRef            string
+	publicKey           string
+	rekorURL            string
+	strict              bool
+	input               string
+	filepath            string
+}{
+	rekorURL: "https://rekor.sigstore.dev/",
+}
```

Comment:
Let's have a version of this struct for `eval` and a version for `verify`

Reply:
Agreed, structs accessible trough `arguments`